arm-dependencies.sls

```diff
@@ -0,0 +1,78 @@
+arm-dependencies:
+  pkg.installed:
+    - pkgs:
+      - g++-aarch64-linux-gnu
+      - g++-arm-linux-gnueabihf
+
+/home/servo/bin:
+  file.directory:
+    - user: servo
+    - group: servo
+    - dir_mode: 755
+    - file_mode: 644
+    
+arm64_links:
+  cmd.wait:
```

Comment:
Actually, we should be use [file.symlink](https://docs.saltstack.com/en/latest/ref/states/all/salt.states.file.html#salt.states.file.symlink) inside a jinja loop for this logic instead. It's easier to see the intention and ensures we know exactly which symlinks are being created (and we get the `creates` behavior for free).